src/pdf_fmt_plug.c

```diff
@@ -40,7 +40,7 @@ john_register_one(&fmt_pdf);
 #define FORMAT_TAG_OLD_LEN  (sizeof(FORMAT_TAG_OLD)-1)
 #define ALGORITHM_NAME      "MD5 SHA2 RC4/AES 32/" ARCH_BITS_STR
 #define BENCHMARK_COMMENT   ""
-#define BENCHMARK_LENGTH    7
+#define BENCHMARK_LENGTH    0x507
```

Comment:
OK. Different costs, reported as such. Please add a comment about that.

Reply:
For now I have reported so for any formats *not* defining them.